packages/notebook-extension/src/celltaglist.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { NotebookPanel } from '@jupyterlab/notebook';
+import React from 'react';
+//import PropTypes from 'prop-types';
+import { ReactWidget, VDomModel, VDomRenderer } from '@jupyterlab/apputils';
+import { CommandRegistry } from '@lumino/commands';
+
+interface IPropsFilterButton {
+  onClick: () => void;
+  sectionLabel: string;
+  buttonLabel: string;
+}
+
+export function FilterButtonComponent(props: IPropsFilterButton) {
```

Comment:
Should we reuse `Button` from `ui-components` instead to ensure consistent styling?

https://github.com/jupyterlab/jupyterlab/blob/897418f0f25ffbbee3aa06a62770a53246823f85/packages/ui-components/src/components/button.tsx#L30C17-L30C23

Reply:
Alternatively, see the next comment about reusing `CommandToolbarButton` or splitting out a `CommandButton` out of it.

Reply:
@krassowski I can use one of the both type of already existing buttons. Do you think that one option is better ?

Reply:
Depends.
- a) `CommandToolbarButton` is a more complete implementation as it received a lot of care to ensure it shows right tooltip, reacts when command changes and can be interacted with by using keyboard as per accessibility requirements, but you would probably need to put some more time in to extract the proposed `CommandButton` out of it
- b) `Button` does not do any of that, just adds a style class, but you should be able to use it easily without any extra work.

(a) seems preferred to me long term, but short term this PR can be merged with (b).

Also, `CommandToolbarButton` is getting refactored in https://github.com/jupyterlab/jupyterlab/pull/15021 so I would hold off any work around it for now.